src/main/cpp/utils/logging/signals/DragonDataLoggerMgr.cpp

```diff
@@ -38,17 +38,10 @@ DragonDataLoggerMgr *DragonDataLoggerMgr::GetInstance()
 
 DragonDataLoggerMgr::DragonDataLoggerMgr() : m_items() //, m_doubleDatalogSignals(), m_boolDatalogSignals(), m_stringDatalogSignals()
 {
-    // auto logFolder = GetLoggingDir();
-    // // frc::DataLogManager::Start(logFolder, CreateLogFileName());
-    // frc::DataLogManager::Start();
-    // frc::DriverStation::StartDataLog(frc::DataLogManager::GetLog());
-    // DragonDataLoggerSignals::GetInstance();
+    auto logFolder = GetLoggingDir();
+    frc::DataLogManager::Start(logFolder, CreateLogFileName());
```

Comment:
Would it be easier to see handle this if we changed the folder when connected to FMS using the DriverStation calls GetMatchType and GetMatchNumber to create a folder such as Qualification1 inside this folder where all of the logs get put?